src/utils/DashboardUtils.tsx

```diff
@@ -205,7 +205,7 @@ const FeedbackStep: Step = {
   texts: [
     'Wir würden uns sehr freuen, wenn du unseren Fragebogen ausfüllst, damit wir unsere Plattform für euch weiter verbessern können.',
     'Der Fragebogen ist anonymisiert und dauert nicht länger als 5 Minuten.',
-    'Falls du differenziertes Feedback an uns richten möchtest, schreibe eine Mail an feedback@lern-fair.de',
+    'Falls du differenziertes Feedback an uns richten möchtest, schreibe eine Mail an support@lern-fair.de',
```

Comment:
Maybe it's worth using an HTML <a> tag for this or something similar in order to make it clickable.

Reply:
Let's not address this detail so short before the launch of the new user app,